src/index.js

```diff
@@ -210,15 +224,13 @@ module.exports				= {
 	assert_type( "has_prototype",	data );
 
 	if ( data.type === "success" ) {
-	    return new Package( data.payload, {
-		"metadata": data.metadata,
-	    });
+	    return new Package( data.payload, {}, data.metadata );
```

Comment:
```suggestion
	    return new Package( data.payload, null, data.metadata );
```

Reply:
This would create an additional variable assignment operation in the constructor.  Though, if we also update the Package constructor to make the type assertion an else statement, then we can avoid all the operations in that type check.

```diff
  if ( opts === null )
      opts = {};
+ else
      assert_type( "has_prototype", opts );
```

Reply:
I pushed that update, so I'll merge this.